src/news-data.html

```diff
@@ -97,7 +97,7 @@
           this._setFailure(false);
           return;
         }
-        this._fetch('/data/articles/' + article.id + '.html',
+        this._fetch('data/articles/' + article.id + '.html',
```

Comment:
This should be part of the master, right?

Reply:
Yes. Updated and re-merged